### DISCUSSION TITLE
ChIP-Seq data in Signac and vice versa

### DISCUSSION THREAD

**Discussion by quanphan08** [stuart-lab/signac#1111] [Q&A] [answered]
Hello all,

I was wondering if anyone has tried to import ChIP-seq data into Signac to utilize the coverageplot function, or exporting pseudo-bulk scATAC data into other ChIP-Seq package such as ChIPseeker to analyze and visualize the data?
What is a good practice to export the scATAC data as pseudo-bulk? Should I use the AverageExpression function from Seurat to create the object for that?
Thank you for your time!

**Comment by timoast** [comment, marked as answer]
> I was wondering if anyone has tried to import ChIP-seq data into Signac to utilize the coverageplot function

You can plot other types of data alongside your single-cell chromatin data in `CoveragePlot()`as long as it's stored as a bigwig file. You can set the `bigwig` parameter in `CoveragePlot()`, or use the `BigwigTrack()` function to plot bigwig tracks by themselves.

> What is a good practice to export the scATAC data as pseudo-bulk? Should I use the AverageExpression function from Seurat to create the object for that?

I'd suggest splitting the BAM file or fragment file according to cell barcode, to create one file per cell type in the single-cell dataset. You can use the `filterbarcodes` function in [sinto](https://github.com/timoast/sinto) to split a BAM file.

**Reply by quanphan08** [reply]
Thank you for the quick reply and the suggestion!
